app/import_functions_test.go

```diff
@@ -3449,9 +3449,9 @@ func TestImportImportEmoji(t *testing.T) {
 	err := th.App.importEmoji(&data, true)
 	assert.NotNil(t, err, "Invalid emoji should have failed dry run")
 
-	emoji, err := th.App.Srv().Store.Emoji().GetByName(*data.Name, true)
+	emoji, gbnErr := th.App.Srv().Store.Emoji().GetByName(*data.Name, true)
```

Comment:
The convention used is "nErr" which means normalized error. Let's use that.

Reply:
Ok, but what if we have 2 or more normalized errors, what is the convention ?

Reply:
Do you mean the case where one normalized error shadows another normalized error ? In that case, we can write `nErr2`, which follows the standard approach to avoid shadowing.